File: app/src/main/java/com/example/files/models/SelfUpdate.java
package com.example.files.models;

import com.google.gson.annotations.SerializedName;

public class SelfUpdate {

    @SerializedName("version_name")
    public String versionName = "";

    @SerializedName("version_code")
    public int versionCode = 0;

    @SerializedName("files_build")
    public String filesBuild = "";

    @SerializedName("changelog")
    public String changelog = "";

}

File: app/src/main/java/com/example/files/models/SelfUpdate.kt
package com.example.files.models

import com.google.gson.annotations.SerializedName

data class SelfUpdate(
    @JvmField
    @SerializedName("version_name")
    val versionName: String = "",

    @JvmField
    @SerializedName("version_code")
    val versionCode: Int = 0,

    @JvmField
    @SerializedName("files_build")
    val filesBuild: String = "",

    @JvmField
    @SerializedName("changelog")
    val changelog: String = ""
)
